### DISCUSSION TITLE
🚀 Docs: Chart of EE version compatiablity with PHP

### DISCUSSION THREAD

**Discussion by blairliikala** [ExpressionEngine/ExpressionEngine#3863] [Feature Requests]
### Suggest an idea for ExpressionEngine

I often see questions of what old version of ExpressionEngine works on what version of PHP.  This isn't clearly stated in the documentation in a unified location, and is scattered between only major version docs.  Also for building add-ons, knowing what version of php each EE version works on helps to target compatability as well.

Solution:
A simple chart or grid that lists version of php with versions of EE.  Similar to php.net and the supported chart.  It shows each php version, and if it works with what version of EE using simple gray/green/yellow/red (pending/works/not recommended/does not work).  Should be limited to EE versions where compatability changes, so not every minor version needs to be listed.  This would help when developers are upgrading older sites, new sites deciding what version to use, and add-on developers targeting EE and PHP compatability.
<img width="307" alt="Screenshot 2023-10-25 at 8 56 50 AM" src="https://github.com/ExpressionEngine/ExpressionEngine/assets/4128963/d3643ba4-96af-4927-b95c-43febf0b4c22">

**Comment by intoeetive** [comment]
@Yulyaswan can you make a draft PR with a table like this for https://docs.expressionengine.com/latest/installation/requirements.html ? I'll fill it with the correct data after

**Reply by Yulyaswan** [reply]
PR - https://github.com/ExpressionEngine/ExpressionEngine-User-Guide/pull/820

**Comment by bryannielsen** [comment]
This has been completed in the Documentation under [System Requirements](https://docs.expressionengine.com/latest/installation/requirements.html#php-version-requirements-across-ee-versions)
